recipes/bdbag/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "bdbag" %}
+{% set version = "1.5.6" %}
+{% set sha256 = "ae057fec9640442ad039fe50b60afa1d71c0109f9163d182f4f4b27774bbc8b3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - bdbag = bdbag.bdbag_cli:main
+    - bdbag-utils = bdbag.bdbag_utils:main
+
+requirements:
+  host:
+    - python >=3
+    - pip
+    - setuptools
+  run:
+    - python >=3
+    - pytz
+    - tzlocal
+    - requests
+    - certifi
+    - bagit
+    - boto3 >=1.9.5
+    - botocore
+    - awscli
+    - globus-sdk >=1.6.0
+
+test:
+  imports:
+    - bdbag
+  commands:
+    bdbag -h
```

I think this is technical valid but it's more conventional to do:
```suggestion
    - bdbag -h
```